src/components/modules/users/ui/pages/PublicProfile.tsx

```
@@ -0,0 +1,166 @@
+"use client";
+
+import { useParams, useRouter } from "next/navigation";
+import { Avatar, AvatarFallback, AvatarImage } from "@/components/ui/avatar";
+import { Button } from "@/components/ui/button";
```

ESLint

[error] 'Button' is defined but never used.

(@typescript-eslint/no-unused-vars)